depends/patches/qt/windows_lto.patch

```diff
@@ -8,7 +8,7 @@ diff --git a/qtbase/src/widgets/accessible/simplewidgets.cpp b/qtbase/src/widget
 index 107fd729fe..0e61878f39 100644
 --- a/qtbase/src/widgets/accessible/simplewidgets.cpp
 +++ b/qtbase/src/widgets/accessible/simplewidgets.cpp
-@@ -109,6 +109,8 @@ QString qt_accHotKey(const QString &text);
```

Comment:
In 9d3be36f758095167ddd762400fda1a3ea76e8f9: This patch was rebased, but testing a Windows build with LTO and this branch, it doesn't seem to compile. So it seems like we should either fixup the patch to make builds under LTO work, or drop the patch and re-add support for LTO later?

Reply:
> ... or drop the patch and re-add support for LTO later?

Agree on that.

Also, from https://bugreports.qt.io/browse/QTBUG-125034:
> LTCG is unfortunately known to be quite broken with MinGW toolchains.
> We don't support LTCG with MinGW but would accept patches if they're not too involved.

Reply:
The patch has been [dropped](https://github.com/bitcoin/bitcoin/pull/30997#issuecomment-2760340414).